packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -0,0 +1,99 @@
+.advertiser-page {
+    &__container {
+        display: flex;
+        flex-flow: column;
+        height: calc(100vh - 47px - 53px - 60px - 52px - 36px - 2.4rem - 1.6rem);
+        min-height: 555px;
+        overflow-y: scroll;
+    }
+
+    &__header {
+        display: inherit;
+        height: fit-content;
+        justify-content: space-between;
+        width: 657px;
+
+        &-avatar {
+            align-items: center;
+            border-radius: 50%;
+            color: var(--text-colored-background);
+            display: flex;
+            font-size: 12px;
+            height: 3.2rem;
+            justify-content: center;
+            margin-right: 8px;
+            width: 3.2rem;
+        }
+
+        &-details {
+            display: flex;
+        }
+
+        &-name {
+            font-size: 16px;
+            height: fit-content;
+            padding-top: 7px;
+            width: fit-content;
+        }
+
+        &-verification {
+            display: flex;
+            font-size: 14px;
+            margin-right: 5px;
+
+            &-address {
+                margin-left: 16px;
+            }
+        }
+    }
+
+    &__stats {
+        border-bottom: none;
+        display: flex;
+        grid-template-columns: 25fr 25fr 25fr 25fr 25fr 25fr;
```

Comment:
you can use `repeat(6, 25fr)`

if you want responsive, can use `repeat(auto-fill, minmax(20rem, 1fr))`

but, 1fr is enough, since 1fr means the fraction from the parents. why 25fr?, do you mean rem?

Reply:
You're right, this should be 1fr, updated this as well.